src/status_im/contexts/chat/messenger/messages/transport/events.cljs

```diff
@@ -170,13 +169,6 @@
                   (models.visibility-status-updates/handle-visibility-status-updates
                    visibility-status-updates-clj)))
 
-      (seq accounts)
-      (do
-        (js-delete response-js "accounts")
-        (rf/merge cofx
-                  (process-next response-js sync-handler)
-                  (wallet/update-wallet-accounts (types/js->clj accounts))))
```

Comment:
on second thoughts, perhaps I have removed too much here, and it's just the wallet cofx I need to remove 🤔

Reply:
Yeah, we probably need to revisit this to move (or create a clone) of the `update-wallet-accounts` method and update the assets (tokens and collectibles) if any new account is added.